src/controllers/helpers/excelSheets/columns.js

```diff
@@ -0,0 +1,182 @@
+const styles = require('./styles');
+
+const specification = {
+  name: {
+    displayName: 'اسم الموظف',
+    headerStyle: styles.column,
+    width: 73,
+  },
+  salary: {
+    displayName: 'راتب الموظف الأساسي',
+    headerStyle: styles.column,
+    width: 120,
+  },
+  wifeAllowance: {
+    displayName: 'علاوة زوجة',
+    headerStyle: styles.column,
+    width: 60,
+
+  },
+  childAllownace: {
+    displayName: 'علاوة أولاد',
+    headerStyle: styles.column,
+    width: 60,
+
+  },
+  allowanceGasoline: {
+    displayName: 'بدل بنزين',
+    headerStyle: styles.column,
+  },
+  allowanceWork: {
+    displayName: 'بدل مهام',
+    headerStyle: styles.column,
+    width: 40,
+
+  },
+  transportation: {
+    displayName: 'مواصلات',
+    headerStyle: styles.column,
+  },
+  allowanceJerusalem: {
```

Comment:
**Indentation**
insert a white line before each styling block
make sure that you apply this at the whole file